modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -36,11 +34,9 @@ endif::[]
 
 You need the following:
 
-* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up.
+* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up. Note that multi-region buckets or containers are not supported.
 * A topic on the origin cluster, which you can use as a Remote Read Replica topic on the remote cluster.
-* A separate remote cluster in the same region as the bucket or container used for the origin cluster.
-** If you use a multi-region bucket/container, you can create the read replica cluster in any region that has that bucket/container.
-** If you use a single-region bucket/container, the remote cluster must be in the same region as the bucket/container.
+* A separate remote cluster. This cluster can be in the same or a different region as the bucket or container used for the origin cluster.
```

Comment:
Our version of [RRR doc for Cloud](https://deploy-preview-942--redpanda-docs-preview.netlify.app/redpanda-cloud/get-started/cluster-types/byoc/remote-read-replicas/#prerequisites) says:

> Prerequisites
> - A BYOC reader cluster in Ready state. This separate reader cluster must exist in the same Redpanda organization and the same cloud provider account and region as the source cluster.

Does this need to be edited? It seems to me that the prereq regarding the region can be changed, but I'm not sure about it the reader cluster having to be in the same RP org and cloud _account_ still.

Reply:
The edit makes sense.

Reply:
To further complicate things, different region is only supported in GCP right now. Not in AWS. RRR is not at all supported in Azure.

Reply:
If it's that deep, maybe we should add a table or something about the 3 different providers.

Reply:
Oh, interesting: thanks @frenchfrywpepper! I've added to [this other PR](https://github.com/redpanda-data/cloud-docs/pull/168/files) about beta/LA features. Please also update that PR with any other info you know about beta/LA features. 

I don't think a table for the providers is necessary here. I'd just note the difference for providers in the Prereqs.